packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -237,126 +169,170 @@ fn eval_file<'s>(scope: &mut v8::HandleScope<'s>, path: &str) -> Result<Value<'s
     let js_source_code = new_js_string(scope, &source_code);
     let mut try_catch = v8::TryCatch::new(scope);
     let script = v8::Script::compile(&mut try_catch, js_source_code, None).ok_or_else(|| {
-        let exception = try_catch.exception().unwrap();
-        let exception_string = exception
-            .to_string(&mut try_catch)
-            .unwrap()
-            .to_rust_string_lossy(&mut try_catch);
+        let exception = read_try_catch_exception(&mut try_catch).unwrap();
```

Comment:
It feels weird that we need to `unwrap` here. Do you see a way to get around this?